### ISSUE TITLE
Create AMANDA read-only accounts

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#4232]
> What application are you using?

Other / Not Sure

> Describe the problem.

Need to create AMANDA read-only accounts for:

Jen D
Lance B
Allyson R

> How soon do you need this?

Soon — This week 

> Is there anything else we should know?

ROW staff can provide training. AMD staff have identified impacts to signalized intersections that we believe can be more timely reviewed and see by giving their staff insight into the AMANDA process and permits.

> Requested By
Ryan M.

Request ID: DTS20-101240

**Comment by SurbhiBakshi** [comment]
@Nadin-Nader @TracyLinder - can you create Amanda accounts, or should this go to CTM? The accounts are needed this week. I have edited the last names on the issue, but if you need that information, please let me know.

**Comment by Nadin-Nader** [comment]
@SurbhiBakshi i can create them but will need them to fill out the form attached. [https://app.zenhub.com/files/140626918/fb8c2b61-39e3-4c56-91c1-12e3ce785668/download](https://app.zenhub.com/files/140626918/fb8c2b61-39e3-4c56-91c1-12e3ce785668/download)

**Comment by SurbhiBakshi** [comment]
@Nadin-Nader - slacked you their full names

**Comment by Nadin-Nader** [comment]
email sent to users:

> Good Afternoon,
> 
> Hope you are all doing well. I am reaching out to you in regards to your AMANDA account requests. Unfortunately, I will need more information than just your names to create the accounts. Furthermore, I also need to make sure you have taken the AMANDA CBT training.
> 
> Please go ahead and have your supervisor fill out the form. Please make sure that they either have the permissions needed or have a similar user whos’ account we can mirror.
> Once I get this form aback as well as the answer to the CBT training, I will create the accounts.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services
> Austin Transportation Department

**Comment by Nadin-Nader** [comment]
Waiting on Lance to provide the NURF forms and let me know that their training is complete


> From: Nader, Nadin 
> Sent: Friday, October 23, 2020 1:51 PM
> To: Ballard, Lance <Lance.Ballard@austintexas.gov>; Duthie, Jen <Jen.Duthie@austintexas.gov>; Richey, Allyson <Allyson.Richey@austintexas.gov>
> Subject: RE: AMANDA Account requests #4232
> 
> Lance,
> 
> Thank you for the quick response. Let me know once the training is complete and you have filled out the forms and I will create the AMANDA accounts. I will put the card/Request on hold for now.
> 
> From: Ballard, Lance <Lance.Ballard@austintexas.gov> 
> Sent: Friday, October 23, 2020 1:24 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Duthie, Jen <Jen.Duthie@austintexas.gov>; Richey, Allyson <Allyson.Richey@austintexas.gov>
> Subject: RE: AMANDA Account requests #4232
> 
> I just requested training access for us. We have not taken it yet.
> 
> Lance Ballard, P.E.
> Mobility Management Center (MMC) Manager
> City of Austin, Austin Transportation Department
> 8700 Cameron Rd
> Austin, TX 78754
> Lance.Ballard@AustinTexas.gov
> 
> From: Nader, Nadin <Nadin.Nader@austintexas.gov> 
> Sent: Friday, October 23, 2020 1:20 PM
> To: Duthie, Jen <Jen.Duthie@austintexas.gov>; Ballard, Lance <Lance.Ballard@austintexas.gov>; Richey, Allyson <Allyson.Richey@austintexas.gov>
> Subject: AMANDA Account requests #4232
> Importance: High
> 
> Good Afternoon,
> 
> Hope you are all doing well. I am reaching out to you in regards to your AMANDA account requests. Unfortunately, I will need more information than just your names to create the accounts. Furthermore, I also need to make sure you have taken the AMANDA CBT training.
> 
> Please go ahead and have your supervisor fill out the form. Please make sure that they either have the permissions needed or have a similar user whos’ account we can mirror.
> Once I get this form aback as well as the answer to the CBT training, I will create the accounts.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services
> Austin Transportation Department

**Comment by Nadin-Nader** [comment]
Received the AMANDA permissions forms from Lance and Allyson but not JEN. I was able to add Lance and Allyson  but their accounts are NOT complete. I still need information from Lance AND Allyson. For Lance, I need his EmployeeID. Allyson; I need her EmployeeID AND Work Phone.
As far as Jen, I never received an AMANDA Permissions for, I emailed the users and asked for the information.

**Comment by Nadin-Nader** [comment]
Response received and sent:

> From: Nader, Nadin 
> Sent: Thursday, October 29, 2020 2:26 PM
> To: Richey, Allyson <Allyson.Richey@austintexas.gov>; Ballard, Lance <Lance.Ballard@austintexas.gov>; Duthie, Jen <Jen.Duthie@austintexas.gov>
> Cc: Linder, Tracy <Tracy.Linder@austintexas.gov>
> Subject: RE: AMANDA Account requests #4232
> 
> Yes, you are correct. Usually contractors do not have EmployeeID’s. So I will go ahead and finish adding your TEST account along with Lance’s for now. Once I get Jen’s info I will add hers. 
> 
> I will set the automation to run tonight and add your PROD accounts. I will be sending an email with login instructions for TEST. Please keep in mind that the instructions work for both TEST and PROD. Also please keep in mind that changing your password on TEST does not affect your PROD login or password.
> 
> From: Richey, Allyson <Allyson.Richey@austintexas.gov> 
> Sent: Thursday, October 29, 2020 2:21 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Ballard, Lance <Lance.Ballard@austintexas.gov>; Duthie, Jen <Jen.Duthie@austintexas.gov>
> Cc: Linder, Tracy <Tracy.Linder@austintexas.gov>
> Subject: Re: AMANDA Account requests #4232
> 
> Good afternoon,
> 
> My phone number is : +1 (512) 974-4075
> 
> Additionally, I do not think I have an employee ID as we're contractors working for the city, not employees directly. 
> 
> Allyson Richey, EIT
> Mobility Management Center (MMC) Manager
> City of Austin, Austin Transportation Department
> 8700 Cameron Rd
> Austin, TX 78754
> Allyson.Richey@AustinTexas.gov
>

**Comment by Nadin-Nader** [comment]
## **Status**
Still waiting on Jens Permissions form

**Comment by Nadin-Nader** [comment]
Created Lance and Allyson's AMANDA Accounts on TEST (Nothing from Jen Yet)and set the automation to create the PROD accounts tonight.

## **Status:* Waiting on Jen to respond and automation to create the other two accounts on TEST.

**Comment by Nadin-Nader** [comment]
Users Lance and Allyson now have PROD accounts. The email sent to Requestors and Brian Craig below:

> Good Morning,
> 
> Hope everyone is doing well. I am reaching out to you in regards to your AMANDA accounts. I have created both Allyson’s and your account (Lance’s). I am not able to create Jen’s account as I have not received the master permissions form for her account.
> 
> Allyson and Lance; please go ahead and try to log into PROD using your USERID’s as the USERNAME and the Password. Please let me know how this goes.
> 
> I have also submitted a request with CTM to have all of you added to the necessary AMANDA Active directory groups. So if you do not have access to attachments, please give it a day or 2 then try again.
> 
> Jen,
> I still have not received your permissions form. In order for me to add your AMANDA Account, I need that AMANDA permissions form with your information sent to me. If not I cannot add your AMANDA Account. Once I get that, I will add your profile to the AMANDA  application.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services
> Austin Transportation Department

## **Status:**
Waiting on Jen still. If i dont hear anything by Tuesday, i will close this out.

**Comment by Nadin-Nader** [comment]
Received Jen's Response with the Permissions form, I created her account on TEST and added the automation to create the account on PROD. I also submitted a request to CTM to add the users to the AMANDA Active Directory **SCTASK0103265**. I sent an email to Jen with Login info.

> Jen,
> 
> Hope you are doing well. I am reaching out to you in regards to your AMANDA account. I have created the account on TEST and set the automation to create your PROD account tonight. I have also submitted a request to CTM to add you, Allyson and Lance to the necessary groups in the Active Directory that AMANDA uses SCTASK0103265. This does not affect your login on either TEST or PROD.
> 
> Please use the information below to log in to your TEST account. (The instructions are the same for TEST and PROD)
> 
> Your UserID is DUTHIEJE
> Your Temporary password is the same as your UserID in all caps.  
> UserID are not case sensitive but your password is.  
> You will be prompted to change it upon login.
> Please keep in mind that changing the password in TEST does not affect your temporary password in PROD. You will have to change you PROD password once the account is created.
>  
> Please log into the AMANDA TEST application to test your account and update your password.  
>  
> In the Future-If need be, 
> you can reset your password using your Employee ID 10142975:
> Steps:
> Click on this link to access the AMANDA Password Reset Application 
> URL:
> https://apps.austintexas.gov/eas/amanda_pw_reset/confirm_user.cfm
> 
> Once you are able to log in, please let me know.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services
> Austin Transportation Department
> 
> 
> 
> From: Duthie, Jen <Jen.Duthie@austintexas.gov> 
> Sent: Saturday, October 31, 2020 7:20 AM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>
> Subject: RE: AMANDA Accounts Created #4232
> 
> Hi Nadin,
> 
> Sorry for the delay. Please find my form attached.
> 
> Thanks,
> Jen
> 
> From: Nader, Nadin <Nadin.Nader@austintexas.gov> 
> Sent: Friday, October 30, 2020 9:50 AM
> To: Ballard, Lance <Lance.Ballard@austintexas.gov>; Richey, Allyson <Allyson.Richey@austintexas.gov>; Duthie, Jen <Jen.Duthie@austintexas.gov>
> Cc: Linder, Tracy <Tracy.Linder@austintexas.gov>; Craig, Brian <Brian.Craig@austintexas.gov>
> Subject: AMANDA Accounts Created #4232
> Importance: High
> 
> Good Morning,
> 
> Hope everyone is doing well. I am reaching out to you in regards to your AMANDA accounts. I have created both Allyson’s and your account (Lance’s). I am not able to create Jen’s account as I have not received the master permissions form for her account.
> 
> Allyson and Lance; please go ahead and try to log into PROD using your USERID’s as the USERNAME and the Password. Please let me know how this goes.
> 
> I have also submitted a request with CTM to have all of you added to the necessary AMANDA Active directory groups. So if you do not have access to attachments, please give it a day or 2 then try again.
> 
> Jen,
> I still have not received your permissions form. In order for me to add your AMANDA Account, I need that AMANDA permissions form with your information sent to me. If not I cannot add your AMANDA Account. Once I get that, I will add your profile to the AMANDA  application.
> 
> Respectfully,
> 
> Nadin A. Nader 
> IT Business Systems Analyst
> Data and Technology Services
> Austin Transportation Department
> 
> 

## **Status:**
Waiting on CTM to add users in Active Directory, The automation to create the account on PROD and for Jen to log in to AMANDA.

**Comment by Nadin-Nader** [comment]
@TracyLinder accounts have been created in TEST and PROD. Closing
